commands/curation/curationaudit.go

```diff
@@ -276,11 +282,17 @@ func (ca *CurationAuditCommand) getAuditParamsByTech(tech coreutils.Technology)
 			SetNpmOverwritePackageLock(true)
 	case coreutils.Maven:
 		ca.AuditParams.SetIsMavenDepTreeInstalled(true)
+	case coreutils.Pip:
+		if ca.PipRequirementsFile() == "" {
+			ca.SetPipRequirementsFile("requirements.txt")
+		}
```

Comment:
```suggestion
		if ca.PipRequirementsFile() == "" {
			ca.SetPipRequirementsFile("setup.py")
		}
```
I think that when this parameter is empty, install try `setup.py` first (using `.`), only if not exists than try to install with `requirements.txt`

Reply:
lets discuss about it, not sure I understand why its better as default.